hrjobcontract/js/src/job-contract/app.js

```diff
@@ -1,8 +1,12 @@
-define([
```

Comment:
Please do not mix auto semistandard changes with manual changes. Not for now, but for future.